package.json

```diff
@@ -14,12 +14,16 @@
     "scripts:admin": "ts-node scripts/admin.ts",
     "scripts:interfaces": "ts-node scripts/interfaces.ts",
     "scripts:poke": "ts-node scripts/poke.ts",
-    "prepare": "husky install"
+    "prepare": "husky install && npm run prepare:abis",
+    "prepare:abis": "./abi/scripts/typechain.sh && ./abi/scripts/tsconfig/tsconfig.sh && ./abi/scripts/typescript/typescript.sh && ./abi/scripts/after-build/after-build.sh"
```

Comment:
I would prefer to use the `scripts:abis` syntax

Is there a reason why this has to be called as a post-install rather than a regular command?

Reply:
I think it would be better if we added the `abi/scripts/` inside the `scripts` folder in the project root

So the path of the typechain.sh script would be `scripts/abi/typechain.sh`

Reply:
Theres not a particular reason behind calling it as a post-install other than to generate the output upon cloning and initiating the repo. It could just be a regular command, would definitely make the installation process cleaner

Also moving the abi scripts to the repo scripts makes a lot of sense yeah